models/stg_marketo__activity_change_data_value.sql

```diff
@@ -17,7 +17,7 @@ with base as (
 ), fields as (
 
     select 
-        activity_date as activity_timestamp,
+        cast(activity_date as {{ dbt_utils.type_timestamp() }}) as activity_timestamp, -- ensure timestamp is passed without timezone in Redshift 
```

Comment:
Would you mind removing the comments from the changes. In general I think casting for timestamp is smart across all warehouses and don't think we need to explicitly call that out in each of these models. Outlining the purpose in the Release Notes and CHANGELOG I think is sufficient.